add #pragma once to stable/library.h

janeyx99 opened the pull request:
This shoulda been there and it was an oversight that it was not! We do not want the same translation unit to process this header multiple times.

Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #154920

janeyx99 committed:
add #pragma once to stable library.h

[ghstack-poisoned]

albanD reviewed (approved):
Empty file should be removed. Good otherwise

Skylion007 commented:
@pytorchbot merge

Skylion007 commented:
@pytorchbot rebase

Skylion007 commented:
Whoops, forgot I approved but empty file was there.

janeyx99 committed:
Update

[ghstack-poisoned]

janeyx99 committed:
Update on "add #pragma once to stable/library.h"

This shoulda been there and it was an oversight that it was not! We do not want the same translation unit to process this header multiple times.




[ghstack-poisoned]
